### ISSUE TITLE
where is impact area needed?

### ISSUE THREAD

**Issue by rnugent3** [HydrologicEngineeringCenter/HEC-FDA#673]
We do a lot of checking for impact area IDs. I need to look at where we expect multiple impact areas and where we don't so that we're not validating redundantly.   

      In general, if we've validated something upstream (like only using one impact area) then I think it's really better to not check for it downstream again. It's likely harmless, but it adds the illusion that there is some possibility that equals might fail, and makes it harder to understand how the code works. 

In short, it just gets confusing. You've got a comment here, which satisfys any concerns, but just wanted to bring that up incase it comes up again.

_Originally posted by @Brennan1994 in https://github.com/HydrologicEngineeringCenter/HEC-FDA/pull/660#discussion_r1034105800_
